sphinx/screens/dashboard/dashboard/src/main/java/chat/sphinx/dashboard/ui/DashboardViewModel.kt

```diff
@@ -899,4 +901,13 @@ internal class DashboardViewModel @Inject constructor(
     override suspend fun onMotionSceneCompletion(value: Any) {
         // Unused
     }
+
+    fun toastIfNetworkConnected(context: Context){
+        if (_networkStateFlow.value is Response.Error) {
+            SphinxToastUtils(true).show(context, R.string.dashboard_network_disconnected_node_toast)
+        }else{
+            SphinxToastUtils(true).show(context, R.string.dashboard_network_connected_node_toast)
+        }
```

Comment:
Use `DashboardSideEffect.Notify` instead of using `SphinxToastUtils` directly on the view model

```suggestion
        DashboardSideEffect.Notify(
            app.getString(
                if (_networkStateFlow.value is Response.Error) {
                    R.string.dashboard_network_disconnected_node_toast
                } else {
                    R.string.dashboard_network_connected_node_toast
                }
            )
        )
```